odmlui/treemodel/TreeIters.py

```diff
@@ -58,18 +58,18 @@ def get_value(self, attr):
         if attr == "value":
             value = self._obj.get_display()
 
-            # Some issues with the rendering of `unicode` in Python 2 directly
-            # to Tree Column cell renderer. Hence, first encode it here.
-            if ValueIter.is_python2:
-                value = value.encode('utf-8')
-
-            # Always escape "&" and "<" since they break assigning values containing
-            # these characters.
-            value = value.replace("&", "&amp;").replace("<", "&lt;")
-
             # If the value is an empty string, render a placeholder text.
-            if value == '':
+            if not value or value == '':
```

Comment:
`not value` also covers `value == ''` so the second condition is redundant.

Reply:
Nice, will change it.